libs/libarchfpga/src/arch_util.h

```diff
@@ -100,4 +100,8 @@ bool pb_type_contains_blif_model(const t_pb_type* pb_type, const std::string& bl
 const t_pin_to_pin_annotation* find_sequential_annotation(const t_pb_type* pb_type, const t_model_ports* port, enum e_pin_to_pin_delay_annotations annot_type);
 const t_pin_to_pin_annotation* find_combinational_annotation(const t_pb_type* pb_type, std::string in_port, std::string out_port);
 
+void link_physical_logical_types(std::vector<t_physical_tile_type>& PhysicalTileTypes,
```

Comment:
It would be great to add comments to these two APIs. I think they are created when adding the support on physical_tile/sub_tiles. I believe that their life cycle is long. Having comments will be easier for other developers.

Reply:
Added a high level comment about this and yes, this function is actually relevant to link the equivalent logical blocks that can be placed at a physical location.

Reply:
Great. Thanks!